### ISSUE TITLE
Selenide does not save files with quotation marks in the file name

### ISSUE THREAD

**Issue by jaroslavdosov** [selenide/selenide#1196]
##  Environment
* **Selenide Version: 5.12.2** 
* **OS Version: windows 10**

## Code To Reproduce Issue
`File file = $(locator).download();`

## Error
`ERROR com.codeborne.selenide.proxy.FileDownloadFilter - Failed to save downloaded file to D:\user\Documents\project\build\downloads\a9a9b0a3-0a22-4d7d-9379-ec4a33b9622b\файл" с "кавычками" да.xlsx for url ... java.io.FileNotFoundException: build\downloads\a9a9b0a3-0a22-4d7d-9379-ec4a33b9622b\файл" с "кавычками" да.xlsx.xlsx (Синтаксическая ошибка в имени файла, имени папки или метке тома)`

## Note
Вut the file is saved as:
D:\user\Documents\project\build\downloads\файл_с_кавычками_да.xlsx

and this directory is empty:
D:\user\Documents\project\build\downloads\a9a9b0a3-0a22-4d7d-9379-ec4a33b9622b\

**Comment by BorisOsipov** [comment]
@jaroslavdosov will be nice to have full error stacktrace

**Comment by jaroslavdosov** [comment]
@BorisOsipov 
For privacy, I replaced ip in some places with characters. For example q.w.e.r
```
[Test worker] INFO com.codeborne.selenide.impl.WebDriverThreadLocalContainer - No webdriver is bound to current thread: 13 - let's create a new webdriver
[Test worker] INFO org.littleshoot.proxy.impl.DefaultHttpProxyServer - Starting proxy at address: 0.0.0.0/0.0.0.0:0
[Test worker] INFO org.littleshoot.proxy.impl.DefaultHttpProxyServer - Proxy listening with TCP transport
[Test worker] INFO org.littleshoot.proxy.impl.DefaultHttpProxyServer - Proxy started at address: /0:0:0:0:0:0:0:0:2138
Starting ChromeDriver 83.0.4103.39 (ccbf011cb2d2b19b506d844400483861342c20cd-refs/branch-heads/4103@{#416}) on port 21878
Only local connections are allowed.
Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.
ChromeDriver was started successfully.
[LittleProxy-0-ClientToProxyWorker-0] WARN com.browserup.bup.mitm.util.SslUtil - Disabling upstream server certificate verification. This will allow attackers to intercept communications with upstream servers.
июн. 10, 2020 1:12:28 PM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: W3C
[Test worker] INFO com.codeborne.selenide.webdriver.WebDriverFactory - BrowserName=chrome Version=83.0.4103.97 Platform=WINDOWS
[Test worker] INFO com.codeborne.selenide.webdriver.WebDriverFactory - Selenide v. 5.12.2
[Test worker] INFO com.codeborne.selenide.webdriver.WebDriverFactory - Selenium WebDriver v. 3.141.59 build time: 2018-11-14T08:17:03
[Test worker] INFO com.codeborne.selenide.drivercommands.CreateDriverCommand - Create webdriver in current thread 13: ChromeDriver -> ChromeDriver: chrome on WINDOWS (5b4591786ffa8bc75489124942ba56af)

 [LittleProxy-0-ProxyToServerWorker-0] WARN com.codeborne.selenide.proxy.ResponseSizeWatchdog - Too large response https://r6---sn-8ph2xajvh-304l.gvt1.com/edgedl/chrome/dict/ru-ru-3-0.bdic?cms_redirect=yes&mh=lx&mip=d.f.g.h&mm=28&mn=sn-8ph2xajvh-304l&ms=nvh&mt=1591779774&mv=u&mvi=5&pl=24&shardbypass=yes: 2309618 bytes

 [LittleProxy-0-ProxyToServerWorker-5] ERROR com.codeborne.selenide.proxy.FileDownloadFilter - Failed to save downloaded file to D:\user\Documents\project\build\downloads\b51eb503-d4d6-4871-b2de-74293a52d814\Файл» с «кавычками» да.xlsx for url http://z.x.w.q/api?fromDate=2020-06-09&toDate=2020-06-10
java.io.FileNotFoundException: build\downloads\b51eb503-d4d6-4871-b2de-74293a52d814\Файл» с «кавычками» да.xlsx (Синтаксическая ошибка в имени файла, имени папки или метке тома)
	at java.base/java.io.FileOutputStream.open0(Native Method)
	at java.base/java.io.FileOutputStream.open(FileOutputStream.java:298)
	at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:237)
	at org.apache.commons.io.FileUtils.openOutputStream(FileUtils.java:365)
	at org.apache.commons.io.FileUtils.writeByteArrayToFile(FileUtils.java:2222)
	at org.apache.commons.io.FileUtils.writeByteArrayToFile(FileUtils.java:2186)
	at org.apache.commons.io.FileUtils.writeByteArrayToFile(FileUtils.java:2171)
	at com.codeborne.selenide.proxy.FileDownloadFilter.filterResponse(FileDownloadFilter.java:88)
	at com.browserup.bup.filters.ResponseFilterAdapter.serverToProxyResponse(ResponseFilterAdapter.java:53)
	at com.browserup.bup.filters.BrowserUpHttpFilterChain.serverToProxyResponse(BrowserUpHttpFilterChain.java:125)
	at org.littleshoot.proxy.impl.ClientToProxyConnection.respond(ClientToProxyConnection.java:418)
	at org.littleshoot.proxy.impl.ProxyToServerConnection.respondWith(ProxyToServerConnection.java:546)
	at org.littleshoot.proxy.impl.ProxyToServerConnection.readHTTPInitial(ProxyToServerConnection.java:259)
	at org.littleshoot.proxy.impl.ProxyToServerConnection.readHTTPInitial(ProxyToServerConnection.java:86)
	at org.littleshoot.proxy.impl.ProxyConnection.readHTTP(ProxyConnection.java:144)
	at org.littleshoot.proxy.impl.ProxyConnection.read(ProxyConnection.java:122)
	at org.littleshoot.proxy.impl.ProxyToServerConnection.read(ProxyToServerConnection.java:229)
	at org.littleshoot.proxy.impl.ProxyConnection.channelRead0(ProxyConnection.java:582)
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:105)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352)
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:287)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352)
	at io.netty.channel.ChannelInboundHandlerAdapter.channelRead(ChannelInboundHandlerAdapter.java:93)
	at org.littleshoot.proxy.impl.ProxyConnection$ResponseReadMonitor.channelRead(ProxyConnection.java:732)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352)
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352)
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352)
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:328)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:302)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352)
	at io.netty.channel.ChannelInboundHandlerAdapter.channelRead(ChannelInboundHandlerAdapter.java:93)
	at org.littleshoot.proxy.impl.ProxyConnection$BytesReadMonitor.channelRead(ProxyConnection.java:686)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1421)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:930)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:163)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:697)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:632)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:549)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:511)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:918)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at java.base/java.lang.Thread.run(Thread.java:834)

Failed to download file {By.id: form-statement-contractor-item} in 10000 ms.Intercepted 1 responses:
  #1  http://z.x.w.q/api?fromDate=2020-06-09&toDate=2020-06-10 -> 200 "OK" {date=Wed, 10 Jun 2020 09:12:39 GMT, server=nginx/1.14.2, content-length=9427, expires=0, x-frame-options=DENY, pragma=no-cache, access-control-allow-origin=*, x-xss-protection=1; mode=block, content-disposition=attachment; filename=%D1%84%D0%B0%D0%B9%D0%BB%22%20%D1%81%20%22%D0%BA%D0%B0%D0%B2%D1%8B%D1%87%D0%BA%D0%B0%D0%BC%D0%B8%22%20%D0%B4%D0%B0xlsx, x-content-type-options=nosniff, content-type=application/vnd.ms-excel, connection=keep-alive, cache-control=no-cache, no-store, max-age=0, must-revalidate, accept-ranges=bytes} application/vnd.ms-excel  (9026 bytes)

java.io.FileNotFoundException: Failed to download file {By.id: form-statement-contractor-item} in 10000 ms.Intercepted 1 responses:
  #1  http://z.x.w.q/api?fromDate=2020-06-09&toDate=2020-06-10 -> 200 "OK" {date=Wed, 10 Jun 2020 09:12:39 GMT, server=nginx/1.14.2, content-length=9427, expires=0, x-frame-options=DENY, pragma=no-cache, access-control-allow-origin=*, x-xss-protection=1; mode=block, content-disposition=attachment; filename=%D1%84%D0%B0%D0%B9%D0%BB%22%20%D1%81%20%22%D0%BA%D0%B0%D0%B2%D1%8B%D1%87%D0%BA%D0%B0%D0%BC%D0%B8%22%20%D0%B4%D0%B0.xlsx, x-content-type-options=nosniff, content-type=application/vnd.ms-excel, connection=keep-alive, cache-control=no-cache, no-store, max-age=0, must-revalidate, accept-ranges=bytes} application/vnd.ms-excel  (9026 bytes)

	at com.codeborne.selenide.impl.DownloadFileWithProxyServer.firstDownloadedFile(DownloadFileWithProxyServer.java:122)
	at com.codeborne.selenide.impl.DownloadFileWithProxyServer.clickAndInterceptFileByProxyServer(DownloadFileWithProxyServer.java:57)
	at com.codeborne.selenide.impl.DownloadFileWithProxyServer.download(DownloadFileWithProxyServer.java:37)
	at com.codeborne.selenide.commands.DownloadFile.execute(DownloadFile.java:54)
	at com.codeborne.selenide.commands.DownloadFile.execute(DownloadFile.java:20)
	at com.codeborne.selenide.commands.Commands.execute(Commands.java:150)
	at com.codeborne.selenide.impl.SelenideElementProxy.dispatchAndRetry(SelenideElementProxy.java:100)
	at com.codeborne.selenide.impl.SelenideElementProxy.invoke(SelenideElementProxy.java:66)
	at com.sun.proxy.$Proxy61.download(Unknown Source)
	at pages.BasePage.download(BasePage.java:115)
	at pages.PersonalAccountPage.downloadFoundedXL(PersonalAccountPage.java:79)
	at tests.jira.PersonalAccountsSearchTest.searchByUPI(PersonalAccountsSearchTest.java:75)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at org.testng.TestRunner.privateRun(TestRunner.java:770)
	at org.testng.TestRunner.run(TestRunner.java:591)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
	at org.testng.SuiteRunner.run(SuiteRunner.java:304)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
	at org.testng.TestNG.runSuites(TestNG.java:1032)
	at org.testng.TestNG.run(TestNG.java:1000)
	at org.gradle.api.internal.tasks.testing.testng.TestNGTestClassProcessor.runTests(TestNGTestClassProcessor.java:141)
	at org.gradle.api.internal.tasks.testing.testng.TestNGTestClassProcessor.stop(TestNGTestClassProcessor.java:90)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.stop(SuiteTestClassProcessor.java:61)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at com.sun.proxy.$Proxy2.stop(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.stop(TestWorker.java:132)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:182)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:164)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:412)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:56)
	at java.base/java.lang.Thread.run(Thread.java:834)

[Test worker] INFO com.codeborne.selenide.drivercommands.CloseDriverCommand - Close webdriver: 13 -> ChromeDriver: chrome on WINDOWS (5b4591786ffa8bc75489124942ba56af)
[Test worker] INFO com.codeborne.selenide.drivercommands.CloseDriverCommand - Close proxy server: 13 -> Selenide proxy server: 2138
[Thread-5] INFO com.codeborne.selenide.drivercommands.CloseDriverCommand - Trying to close the browser ChromeDriver ...
[LittleProxy-0-ClientToProxyWorker-0] INFO org.littleshoot.proxy.impl.ClientToProxyConnection - (AWAITING_INITIAL) [id: 0x67bb63a8, L:/q.w.e.r:2138 - R:/q.w.e.r:2145]: An IOException occurred on ClientToProxyConnection: Удаленный хост принудительно разорвал существующее подключение
[LittleProxy-0-ClientToProxyWorker-7] INFO org.littleshoot.proxy.impl.ClientToProxyConnection - (AWAITING_INITIAL) [id: 0x61c981a9, L:/q.w.e.r:2138 - R:/q.w.e.r:2168]: An IOException occurred on ClientToProxyConnection: Удаленный хост принудительно разорвал существующее подключение
[LittleProxy-0-ClientToProxyWorker-3] INFO org.littleshoot.proxy.impl.ClientToProxyConnection - (AWAITING_INITIAL) [id: 0x0f024167, L:/q.w.e.r:2138 - R:/q.w.e.r:2157]: An IOException occurred on ClientToProxyConnection: Удаленный хост принудительно разорвал существующее подключение
[LittleProxy-0-ClientToProxyWorker-6] INFO org.littleshoot.proxy.impl.ClientToProxyConnection - (AWAITING_INITIAL) [id: 0xef593add, L:/q.w.e.r:2138 - R:/q.w.e.r:2164]: An IOException occurred on ClientToProxyConnection: Удаленный хост принудительно разорвал существующее подключение
[LittleProxy-0-ClientToProxyWorker-5] INFO org.littleshoot.proxy.impl.ClientToProxyConnection - (AWAITING_INITIAL) [id: 0x8e61b395, L:/q.w.e.r:2138 - R:/q.w.e.r:2162]: An IOException occurred on ClientToProxyConnection: Удаленный хост принудительно разорвал существующее подключение
[LittleProxy-0-ClientToProxyWorker-4] INFO org.littleshoot.proxy.impl.ClientToProxyConnection - (AWAITING_INITIAL) [id: 0x983c387a, L:/q.w.e.r:2138 - R:/q.w.e.r:2158]: An IOException occurred on ClientToProxyConnection: Удаленный хост принудительно разорвал существующее подключение
[LittleProxy-0-ClientToProxyWorker-2] INFO org.littleshoot.proxy.impl.ClientToProxyConnection - (AWAITING_INITIAL) [id: 0x960b6deb, L:/q.w.e.r:2138 - R:/q.w.e.r:2155]: An IOException occurred on ClientToProxyConnection: Удаленный хост принудительно разорвал существующее подключение
[LittleProxy-0-ClientToProxyWorker-0] INFO org.littleshoot.proxy.impl.ClientToProxyConnection - (AWAITING_INITIAL) [id: 0x2e657da7, L:/q.w.e.r:2138 - R:/q.w.e.r:2170]: An IOException occurred on ClientToProxyConnection: Удаленный хост принудительно разорвал существующее подключение
[LittleProxy-0-ClientToProxyWorker-1] INFO org.littleshoot.proxy.impl.ClientToProxyConnection - (AWAITING_INITIAL) [id: 0xeeb2820c, L:/q.w.e.r:2138 - R:/q.w.e.r:2154]: An IOException occurred on ClientToProxyConnection: Удаленный хост принудительно разорвал существующее подключение
[Thread-5] INFO com.codeborne.selenide.drivercommands.CloseDriverCommand - Trying to shutdown Selenide proxy server: 2138 ...
[Thread-5] INFO org.littleshoot.proxy.impl.DefaultHttpProxyServer - Shutting down proxy server immediately (non-graceful)
[Thread-5] INFO org.littleshoot.proxy.impl.DefaultHttpProxyServer - Closing all channels (non-graceful)
[Thread-5] INFO org.littleshoot.proxy.impl.ServerGroup - Shutting down server group event loops (non-graceful)
[Thread-5] INFO org.littleshoot.proxy.impl.DefaultHttpProxyServer - Done shutting down proxy server
[Test worker] INFO com.codeborne.selenide.drivercommands.CloseDriverCommand - Closed webdriver 13 in 765 ms
D:\user\Documents\project\src\test\resources\db\deleteCreatingRecord.txt


Chrome > Dynamic > tests.jira.PersonalAccountsSearchTest > searchByUPI FAILED
    java.io.FileNotFoundException at PersonalAccountsSearchTest.java:75



[Thread-3] INFO com.codeborne.selenide.drivercommands.CloseDriverCommand - Close webdriver: 30 -> ChromeDriver: chrome on WINDOWS (null)
[Thread-3] INFO com.codeborne.selenide.drivercommands.CloseDriverCommand - Close proxy server: 30 -> Selenide proxy server: 2138
[Thread-6] INFO com.codeborne.selenide.drivercommands.CloseDriverCommand - Trying to close the browser ChromeDriver ...
[Thread-6] INFO com.codeborne.selenide.drivercommands.CloseDriverCommand - Trying to shutdown Selenide proxy server: 2138 ...
[Thread-3] INFO com.codeborne.selenide.drivercommands.CloseDriverCommand - Closed webdriver 30 in 1 ms
[Webdrivers killer thread] INFO com.codeborne.selenide.impl.UnusedWebdriversCleanupThread - Thread 13 is dead. Let's close webdriver null
[Webdrivers killer thread] INFO com.codeborne.selenide.impl.UnusedWebdriversCleanupThread - No webdriver found for thread: 13 - nothing to close
```

**Comment by KirillFurtikov** [comment]
It's not a bug. You can't save a file not only in selenium, but also create such a file. Such problems need to be solved at the application level, not in tests. The browser automatically removes invalid characters, but selenium does not, but it's not a bug.

**Comment by asolntsev** [comment]
@KirillFurtikov I agree that it's not a bug. This is not a valid file name with quotes. :)

But if real browser do delete quotes, then Selenide could also delete. It would be convenient for Selenide users.

**Comment by asolntsev** [comment]
Hah, it seems that this problem exists only on Windows. 
Linux and Mac allows creating files with quotes in name (see https://github.com/selenide/selenide/tree/download-file-with-quotes).

**Comment by asolntsev** [comment]
Fixed by #1199
